web-client/scripts/build.sh

```diff
@@ -29,47 +51,68 @@ function generate() {
     BINDGEN_OUTPUT="$OUT_DIR/index_bg.wasm"
     OPT_OUTPUT="$BINDGEN_OUTPUT.opt"
 
-    printf "Generating $TYPE WASM for target $TARGET...\n"
+    echo "Generating $TYPE WASM for target $TARGET...\n"
 
     # Generate JS bindings (also generates a GC'd WASM file)
     wasm-bindgen --weak-refs --target $TARGET --out-name index --out-dir $OUT_DIR --no-typescript $CARGO_OUTPUT
```

Comment:
Here and below, needs double quotes around strings involving variables such as `$CARGO_OUTPUT`.